### ISSUE TITLE
Is there a version that works with MacOS?

### ISSUE THREAD

**Issue by AstroOrbis** [unitycoder/UnityLauncherPro#25]

**Comment by unitycoder** [comment]
haven't tested if the existing code would work in mac visual studio.. (at least some of the paths are currently windows only).

*ok i guess thats a no go, i'm using wpf:
"Visual Studio for Mac does not support Windows client projects like Windows Forms, WPF, or UWP."
https://docs.microsoft.com/en-us/visualstudio/productinfo/vs2019-compatibility-mac

**Comment by AstroOrbis** [comment]
Would it be possible to make a MacOS version?

**Comment by unitycoder** [comment]
yes, pretty sure it should be possible (for someone who is familiar with macOS, to know where unity recent project list is saved, how to scan unity installs etc.)

actually there is at least one alternative here for macos,
https://github.com/unitycoder/UnityLauncherPro/wiki/Alternative-Launchers

**Comment by AstroOrbis** [comment]
Unity installs are all stored in `/Applications/Unity/(versionIdentifier)/(files)`
Projects are in ~/Unity I believe

**Comment by unitycoder** [comment]
uno platform: https://platform.uno/

comments on winforms, wpf,
![image](https://user-images.githubusercontent.com/5438317/153723844-77f25031-95ae-40cd-8668-1535f915d616.png)
https://twitter.com/reillywood/status/1485485529206382592

**Comment by unitycoder** [comment]
also Unity could be used to build multiplatform launcher.. and already have all the c# code here,
but just afraid that the UI would be quite limited then.. (not fun to build datagrid or other components from scratch..)

**Comment by AstroOrbis** [comment]
It's been a year lmao

MacOS default UI looks pretty good, and they're more menus than I can count on two fingers in binary to play with

**Comment by unitycoder** [comment]
this is now available in VS2022, need to check how it works:
https://devblogs.microsoft.com/dotnet/dotnet-maui-visualstudio-2022-release/
